tests/numpy/test_arithmetic.py

```diff
@@ -144,9 +144,78 @@ def test_inner_outer(nps_app_inst):
     assert np.allclose(np.inner(A, B), nps.inner(nps_A, nps_B).get())
     assert np.allclose(np.outer(A, B), nps.outer(nps_A, nps_B).get())
 
+# TODO (hme): Add broadcast tests.
 
-# TODO(hme): Add broadcast tests.
+def test_tensordot_shape_error(nps_app_inst):
+    import numpy as np
+    from nums.core.array.utils import is_array_like
+    from nums import numpy as nps
+    import pytest
+    assert nps_app_inst is not None
 
+    def check_simple_shape_mismatch_error(_np_a, _np_b):
```

Comment:
Do you remember why we added this? Since this is not testing tensordot operations, I think we can remove this.